cmd/tailscale/cli/funnel.go

```diff
@@ -33,14 +33,32 @@ func newFunnelCommand(e *serveEnv) *ffcli.Command {
 		ShortUsage: strings.TrimSpace(`
 funnel <serve-port> {on|off}
   funnel status [--json]
+  funnel https:<port> <mount-point> <source> [off]
```

Comment:
why do you need the source for `off` mode?

Reply:
We don't; you can give just the mount point.

Reply:
But in the terminal if you do `<up> off` it'll work